app/allocations/controllers/list.js

```diff
@@ -0,0 +1,12 @@
+const presenters = require('../../../common/presenters')
+
+module.exports = function list(req, res) {
+  const { allocations = [] } = res.locals
+  const template = 'allocations/views/list'
+  const locals = {
+    pageTitle: req.t('allocations::dashboard.heading'),
+    ...presenters.allocationsToTable(allocations),
```

Comment:
It might be worth returning this as a specific variable to the view rather than just `head` and `rows`. I've seen some evolution of some of Andy's designs that display multiple tables on a page.

Feels easier to read the template too if the variable clearly explains the data being loaded into the table.